src/main/scala/tech/cryptonomic/nautilus/cloud/domain/metering/stats/MeteringStatsRepository.scala

```diff
@@ -14,6 +16,6 @@ trait MeteringStatsRepository[F[_]] {
   def getLastStats(list: List[UserId]): F[List[AggregatedMeteringStats]]
 
   /** Fetches stats for the given user */
-  def getStatsPerUser(userId: UserId): F[List[AggregatedMeteringStats]]
+  def getStatsPerUser(userId: UserId, from: Option[Instant]): F[List[AggregatedMeteringStats]]
```

Comment:
Would it be useful to put a default value of `None` here, and replicate it to any implementation?